src/extension.ts

```diff
@@ -434,23 +434,29 @@ function handleLogViaKubectlCli(podName: string) : Promise<string> {
 async function registerTutorialWithDidact(context: vscode.ExtensionContext) {
 	// call didact command to register tutorial
 	try {
-		// command ID: vscode.didact.register
-		const commandId = 'vscode.didact.register';
-
-		// then pass name, uri, and category
-		const tutorialName = 'Your First Integration';
-		const tutorialPath = path.join(context.extensionPath, './didact/camelk/first-integration.md');
-		const tutorialUri = vscode.Uri.parse(`file://${tutorialPath}`);
-		const tutorialCategory = 'Apache Camel K';
-
-		console.log('Tutorial URI registered: ' + tutorialUri.fsPath);
-
-		await vscode.commands.executeCommand(
-			commandId,
-			tutorialName, 
-			tutorialUri,
-			tutorialCategory);
-
+		// test to ensure didact is available 
```

Comment:
I was trying to do everything in this one method to avoid complicating things for a developer simply seeking to register a new tutorial with the system.

Reply:
I think that it is simpler to have explicit naming with method names than several comments to explain what is the purpose of the code.

to quote Ron Jeffries author of Extreme Programming Installed and Extreme Programming Adventures in C#

Expressiveness to me includes meaningful names, and I am likely to change the names of things several times before I settle in. With modern coding tools such as Eclipse, renaming is quite inexpensive, so it doesn’t trouble me to change. Expressiveness goes beyond names, however. I also look at whether an object or method is doing more than one thing. If it’s an object, it probably needs to be broken into two or more objects. If it’s a method, I **will always use the Extract Method refactoring** on it, **resulting in one method that says more clearly what it does, and some submethods saying how it is done.**